RequestManagementSystem/Client/Operation.py

```diff
@@ -201,12 +201,12 @@ def __len__( self ):
   @property
   def sourceSEList( self ):
     """ helper property returning source SEs as a list"""
-    return self.SourceSE.split( "," )
+    return self.SourceSE.split( "," ) if self.SourceSE else ['']
```

Comment:
I know, this was the old behavior. There are places where we do Operation.targetSEList[0], which certainly not the right thing to do, but this requires much more digging that can't be done now. However this fix is necessary for the RMS to execute some operation, it is hotfixed